DynamicActiveRecord.php

```diff
@@ -257,10 +248,11 @@ public function __set($name, $value)
         try {
             parent::__set($name, $value);
         } catch (UnknownPropertyException $ignore) {
-            if (!preg_match('{^[a-z_\x7f-\xff][a-z0-9_\x7f-\xff]*$}i', $name)) {
+            if (!preg_match('{^[a-z_\x7f-\xff][a-z0-9_.\x7f-\xff]*$}i', $name)) {
                 throw new InvalidCallException('Invalid attribute name "' . $name . '"');
             }
-            $this->dynamicAttributes[$name] = $value;
+
+            $this->setAttribute($name, $value);
```

Comment:
this change brakes `$model->setAttribute('strWithoutDot', something)`

you could do something like:

    if there is . in $name
        call setAttribute
    or else
        write to $this->dynamicAttributes[$name]